src/ontology/mondo-edit.obo

```diff
@@ -328543,6 +328543,7 @@ id: MONDO:0014269
 name: combined oxidative phosphorylation deficiency 19
 def: "Any combined oxidative phosphorylation deficiency in which the cause of the disease is a mutation in the LYRM4 gene." [MONDO:patterns/disease_series_by_gene]
 subset: gard_rare {source="GARD:18466", source="MONDO:GARD"}
+subset: ordo_disorder {source="Orphanet:397593"}
```

Comment:
@twhetzel was there a reason to add this subset annotation manually?

Reply:
I see, it is because you removed it from the obsoleted term that was also equivalent to Orphanet:397593.
It is fine. It will be updated automatically, but it is fine to add it manually here.